packages/backend/src/services/UserService.ts

```diff
@@ -397,6 +397,7 @@ export class UserService {
             });
 
             if (
+                lightdashConfig.groups.enabled === true &&
```

Comment:
if groups are not enabled we should not group sync even tho enableGroupSync might be present